Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:edge
```

Comment:
Should we use our internal tagged alpine image? Or does alpine themselves provide any tags besides "edge"? We've been bitten a couple times now by base images changing in ways that broke our images, so I feel like we should avoid relying on third-party "latest"/"edge" images.